Pod/VOKNavigationHelper/VOKNavigationHelper.m

```diff
@@ -0,0 +1,51 @@
+//
+//  VOKNavigationHelper.m
+//  VOKUtilities
+//
+//  Created by Brock Boland on 1/24/17.
+//  Copyright (c) 2017 Vokal. All rights reserved.
+//
+
+#import "VOKNavigationHelper.h"
+
+@implementation VOKNavigationHelper
+
++ (void)clearExistingViewsAndSwitchToViewController:(UIViewController *)viewController
+{
+    // Enforce the non-null
+    if (viewController == nil) {
+        return;
+    }
+
+    id<UIApplicationDelegate> appDelegate = [UIApplication sharedApplication].delegate;
+    
+    // Get rid of old VC's and views
+    [appDelegate.window.subviews makeObjectsPerformSelector:@selector(removeFromSuperview)];
+    
+    if (appDelegate.window.rootViewController.presentedViewController) {
+        [appDelegate.window.rootViewController dismissViewControllerAnimated:NO
+                                                                  completion:^{
+                                                                      appDelegate.window.rootViewController = viewController;
+                                                                  }];
+    } else {
+        appDelegate.window.rootViewController = viewController;
```

Comment:
It might be worthwhile to put this line inside a local-variable block and pass that block as the animation completion above and call that block manually here, to avoid the repeated line.